cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -596,7 +593,7 @@
 	      <t>Claim Description: Expiration Time</t>
 	      <t>JWT Claim Name: <spanx style="verb">exp</spanx></t>
 	      <t>CBOR Key Value: 4</t>
-	      <t>CBOR Major Type: 6, tag value 1</t>
+	      <t>CBOR Major Type: 6.1</t>
```

Comment:
I think these lines should actually say "CBOR tag 6.1 followed by CBOR number (major types 0, 1, or 7)".